contrib/cirrus/integration_test.sh

```diff
@@ -36,6 +36,7 @@ else
     make
     make install PREFIX=/usr ETCDIR=/etc
     make test-binaries
+    make install.tools
```

Comment:
That should probably happen from setup_environment.sh no?

Reply:
seems ok here? we do a make of the test-binaries?